composer.json

```diff
@@ -29,12 +29,12 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "roave/infection-static-analysis-plugin": "^1.6",
+        "roave/infection-static-analysis-plugin": "^1.7",
         "spatie/phpunit-watcher": "^1.23",
-        "vimeo/psalm": "^4.3",
+        "vimeo/psalm": "^4.7",
         "yiisoft/aliases": "^1.1",
-        "yiisoft/di": "^3.0@dev",
-        "yiisoft/event-dispatcher": "^1.0"
+        "yiisoft/event-dispatcher": "^1.0",
```

Comment:
```suggestion
```
I see you also replaced the dispatcher in tests.
Maybe he is no longer needed either?

Reply:
Yes, `yiisoft/event-dispatcher` not need for tests, but I can't remove him, because composer required `"psr/event-dispatcher-implementation": "1.0.0",`.

Reply:
https://packagist.org/packages/yiisoft/psr-dummy-provider